src/libs/antares/study/parameters.cpp

```diff
@@ -345,7 +345,7 @@ void Parameters::reset()
 
     // Shedding strategies
     power.fluctuations = lssFreeModulations;
-    shedding.policy = shpShavePeaks;
+    shedding.policy = shpAccurateShavePeaks;
```

Comment:
@flomnes 
Are there some precautions to take when changing the default value of a parameter like this ?

Reply:
It will change the results. But the whole point of this PR is to change the results.

More seriously, the "shave peaks" algorithm generates non-conforming results, so we'd best avoid using it (deprecated property).